### PR TITLE
refactor(components): toggleGroup creation from useToggleGroup

### PR DESCRIPTION
**Pull request by jerader**
closes AUTH-650

# Overview

splits up the styling from `useToggleGroup` out into `ToggleGroup` into the components directory. Couldn't move `useToggleGroup` into components since it has the analytics hook embedded into it. When it is used in PD we can create our own hook.

## Test Plan and Hands on Testing

look at `ToggleGroup` in storybook and see that it works as expected. also review the code

## Changelog

- move styling from `useToggleGroup` out into its own component and make a story

## Risk assessment

low

### CODE REVIEW

File: components/src/atoms/ToggleGroup/index.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+const BUTTON_GROUP_STYLES = css`
+  border-radius: ${BORDERS.borderRadius8};
+  margin-top: -1px;
+  width: fit-content;
+
+  button {
+    height: 28px;
+    width: auto;
+    font-weight: 400;
```
Comment (koji) [inline components/src/atoms/ToggleGroup/index.tsx]:
```suggestion
    font-weight: ${TYPOGRAPHY.fontWeightRegular};
```

File: components/src/atoms/ToggleGroup/index.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  )
+}
+
+const BUTTON_GROUP_STYLES = css`
+  border-radius: ${BORDERS.borderRadius8};
+  margin-top: -1px;
+  width: fit-content;
+
+  button {
+    height: 28px;
```
Comment (koji) [inline components/src/atoms/ToggleGroup/index.tsx]:
```suggestion
    height: 1.75rem;
```

File: components/src/atoms/ToggleGroup/index.tsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    width: auto;
+    font-weight: 400;
+    font-size: 11px;
+    line-height: 14px;
+    box-shadow: none;
+    padding-top: 6px;
+    padding-bottom: 8px;
```
Comment (koji) [inline components/src/atoms/ToggleGroup/index.tsx]:
```suggestion
    padding-bottom: ${SPACING.spacing8};
```

File: components/src/atoms/ToggleGroup/index.tsx
Comment (koji) [inline components/src/atoms/ToggleGroup/index.tsx:1]: if you can add test for `ToggleGroup`, it will be great.

**Review by ncdiehl11 — APPROVED** [review]
Looks good, but one question— do we want to expose props for left/rightDisabled? I see styles for it, but no way to set disabled state.

**Comment by jerader** [comment]
> Looks good, but one question— do we want to expose props for left/rightDisabled? I see styles for it, but no way to set disabled state.

@ncdiehl11 good question, i just copied and pasted the styling that existed and didn't realize that is in there, looks like it was added 2 years ago so maybe we have removed that functionality. I'll delete it!

**Pull request merged**